uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -1227,6 +1227,119 @@ Feature: GGMQ-1
+      | aws.greengrass.clientdevices.Auth        | LATEST                                  |
+      | aws.greengrass.clientdevices.mqtt.EMQX   | LATEST                                  |
+      | aws.greengrass.clientdevices.IPDetector  | LATEST                                  |
+      | aws.greengrass.clientdevices.mqtt.Bridge | LATEST                                  |
+      | <agent>                                  | classpath:/local-store/recipes/<recipe> |
+    And I create client device "large_payload_publisher"
+    When I associate "large_payload_publisher" with ggc
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.Auth configuration to:
+    """
+{
+    "MERGE":{
+        "deviceGroups":{
+            "formatVersion":"2021-03-05",
+            "definitions":{
+                "MyPermissiveDeviceGroup":{
+                    "selectionRule":"thingName: ${large_payload_publisher}",
+                    "policyName":"MyPermissivePolicy"
+                }
+            },
+            "policies":{
+                "MyPermissivePolicy":{
+                    "AllowAll":{
+                        "statementDescription":"Allow client devices to perform all actions.",
+                        "operations":[
+                            "*"
+                        ],
+                        "resources":[
+                            "*"
+                        ]
+                    }
+                }
+            }
+        }
+    }
+}
+    """
+    And I update my Greengrass deployment configuration, setting the component <agent> configuration to:
+    """
+{
+    "MERGE":{
+        "controlAddresses":"${mqttControlAddresses}",
+        "controlPort":"${mqttControlPort}"
+    }
+}
+    """
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.mqtt.Bridge configuration to:
+    """
+{
+    "MERGE":{
+        "mqttTopicMapping":{
+            "mapping1:":{
+                "topic":"${large_payload_publisher}topic/to/iotcore",
+                "source":"LocalMqtt",
+                "target":"IotCore"
+            }
+        }
+    }
+}
+    """
+    And I deploy the Greengrass deployment configuration
+    Then the Greengrass deployment is COMPLETED on the device after 5 minutes
+    And the aws.greengrass.clientdevices.mqtt.EMQX log on the device contains the line "is running now!." within 1 minutes
+
+    And I discover core device broker as "default_broker" from "large_payload_publisher" in OTF
+    And I connect device "large_payload_publisher" on <agent> to "default_broker" using mqtt "<mqtt-v>"
+
+    And I subscribe "large_payload_publisher" to "${large_payload_publisher}topic/to/iotcore" with qos 1
+    When I publish from "large_payload_publisher" to "${large_payload_publisher}topic/to/iotcore" with qos 1 and large message with beginning of "Hello world1" with length 130098
```

Comment:
can we include original comment, otherwise 130098 will be confusing

```
# 130098 is 74 bytes short of 128KiB. However, the Moquette limit is frame size, not payload size.
# We should address this, but for now, we'll just decrease the payload slightly.
# NOTE: 74 bytes include a fixed header + variable length header which includes the topic name
```

Reply:
added in dfa284f5554966315e8154577cbbf4f3de7d068f